src/tx-builder/services/DydxGovernance.ts

```diff
@@ -631,4 +633,18 @@ export default class DydxGovernanceService extends BaseService<DydxGovernor> {
       topAgainstVotes,
     };
   }
+
+  public async getGovernanceVoters(
+    endBlock: number,
+    startBlock: number = DEPLOYMENT_BLOCK,
+  ): Promise<Set<string>> {
+    const governor: DydxGovernor = this.getContractInstance(
+      this.dydxGovernanceAddress,
+    );
+    const filter = governor.filters.VoteEmitted(null, null, null, null);
+    const events = await governor.queryFilter(filter, startBlock, endBlock);
+
+    // args[1] = ethereumAddress
```

Comment:
nit: Mind pasting the whole event signature (so that people can see the first arg is the voters address)?